database/src/pool/postgres.rs

```diff
@@ -285,6 +286,22 @@ static MIGRATIONS: &[&str] = &[
     alter table pstat_series drop constraint test_case;
     alter table pstat_series add constraint test_case UNIQUE(crate, profile, scenario, backend, target, metric);
     "#,
+    r#"
+    CREATE TABLE IF NOT EXISTS benchmark_requests (
```

Comment:
All our other tables use a singular name, so it should be `benchmark_request`. Sorry for naming it wrong in the document :)